test/kwtool/crawlers/schemas/keyword_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Kwtool.Crawlers.Schemas.UserTest do
+  use Kwtool.DataCase, async: true
+
+  alias Kwtool.Crawlers.Schemas.Keyword
+
+  describe "keyword/2" do
+    test "returns valid keyword changeset when given valid attributes" do
+      keyword_params = %{phrase: "elixir", status: 1, user_id: 1}
+      changeset = Keyword.changeset(keyword_params)
+
+      assert changeset.valid?
+      assert changeset.changes.phrase == "elixir"
+    end
+
+    test "returns invalid when given valid attributes" do
+      keyword_params = %{phrase: "elixir", status: 1}
+      changeset = Keyword.changeset(keyword_params)
+
+      refute changeset.valid?
```

Comment:
Probably check the changeset errors here, we can do

```
assert errors_on(changeset) == %{...}
```

So we know which errors are we expecting. Normally in the project we will do the test for required attributes in one case and then other validation if has.

For example this one we can write as

```elixir
test "returns invalid when required attributes are missing" do
  changeset = Keyword.changeset(%{})

  refute changeset.valid?

  assert errors_on(changeset) == %{
               phrase: ["can't be blank"],
               status: ["can't be blank"],
               user_id: ["can't be blank"],
  }
end
```

Reply:
Updated! Thanks 👍